pkg/readiness/ready_tracker.go

```diff
@@ -160,7 +160,7 @@ func (t *Tracker) For(gvk schema.GroupVersionKind) Expectations {
 			return t.assignImage
 		}
 		return noopExpectations{}
-	case gvk.GroupVersion() == expansionv1alpha1.GroupVersion && gvk.Kind == "ExpansionTemplate":
+	case gvk.Group == expansionv1alpha1.GroupVersion.Group && gvk.Kind == "ExpansionTemplate":
```

Comment:
I thought we changed this in a separate PR 👀 ; trying to dig that up

Reply:
https://github.com/open-policy-agent/gatekeeper/pull/2635 changed it for all the other types besides `ExpansionTemplate`, which did not exist in the tracker at the time.

Then, when we added expansion status, I forgot to change ET to not use `Version` :)